user-manual/administer/dashboard-admin.rst

```diff
@@ -559,46 +558,83 @@ Clicking the date, name or UUID will display a report of the failure:
 
-.. _dashboard-transfer-source:
+.. _dashboard-storage-locations:
 
-Transfer source location
-------------------------
+Storage locations
+-----------------
 
-Archivematica allows you to start transfers using the operating system's file
-browser or via a web interface. Source files for transfers, however, cannot be
-uploaded using the web interface; they must exist on volumes accessible to the
-Archivematica MCP server and configured via the Storage Service.
+This section of the Administration page displays storage locations that have
+been configured for the Archivematica instance. Storage locations are configured
+in the :ref:`Storage Service <storageService:administrators>`. This table
+presents information from the Storage Service.
 
-When starting a transfer you are required to select one or more directories of
-files to add to the transfer.
+.. image:: images/dashboard-storage-locations.*
+   :align: center
+   :width: 80%
+   :alt: Table showing available storage locations configured for the Archivematica pipeline
 
-.. _dashboard-AIP-stor:
+The table on the Storage locations page displays the :ref:`purpose
+<location-purposes>` of the storage location, the location's description, how
+much space has been used, and the path.
 
-AIP storage locations
----------------------
+If a quota has not been set for the storage location, the location's available
+space will be set to *unlimited*. Quotas can be set for some types of locations
+through the :ref:`Storage Service <storageService:locations>`.
 
-AIP storage directories are directories in which completed AIPs are stored.
-Storage directories can be specified in a manner similar to transfer source
-directories using the Storage Service.
+*Used / available* values are not currently calculated for all types of storage
+locations, such as Replicator, Transfer Backlog, and Transfer Source locations.
 
-You can view your transfer source directories in the Administrative tab of the
-dashboard under "AIP storage locations".
+Depending on the configuration of your Archivematica deployment, you may not see
+values for all storage locations.
 
 .. _dashboard-usage:
 
 Processing storage usage
 ------------------------
 
-This section of the Administration page displays various processing locations
-with their current usage of available space.
+This section of the Administration page displays the Archivematica instance's
+processing locations. Processing locations are used to hold materials while they
+arebeing processed, as well as temporary storage directories for certain types
+of content - for example, failed and rejected transfers.
+
+.. important::
+
+   This page calculates usage statistics for Archivematica instances that have
+   been deployed with a shared directory on the same disk as the Archivematica
+   instance. If your Archivematica instance has been configured a different way,
+   the usage statistics may not be displayed.
```

Comment:
This is a bit confusing for me. If the AM pipeline doesn't have access to the shared directory many things will go wrong (see https://github.com/artefactual/archivematica/issues/885#issuecomment-551179239).

Reply:
I guess what I was trying to communicate was that this anticipates a fairly standard setup; if an AM instance is set up way off spec, it might not work. Maybe it's best to just remove the admonition.